version: "3.9"
services:
  test:
    image: "exercise-01"
    volumes:
      - .:/home/syncopy/funlab/de/app
    command: python3 -m pytest
  run:
    image: "exercise-01"
    volumes:
      - .:/home/syncopy/funlab/de/app
    command: python3 main.py